pyproject.toml

```diff
@@ -222,12 +222,6 @@ precision = 1
 sort = "cover"
 show_missing = true
 
-[tool.nbqa.mutate]
-pyupgrade = 1
```

Comment:
A note to self, the reason this is safe to remove is that the option was deprecated in `v1.0.0`. c.f. https://github.com/nbQA-dev/nbQA/blob/26f3ce04cd29bdfcda487fefdbae78925e80a5ff/nbqa/cmdline.py#L37